perceval/runtime/job_status.py

```diff
@@ -143,6 +143,10 @@ def success(self):
     def failed(self):
         return self._status in [RunningStatus.CANCELED, RunningStatus.ERROR]
 
+    @property
+    def maybe_completed(self):
+        return self._status in [RunningStatus.PARTIAL_COMPLETION, RunningStatus.UNKNOWN]
```

Comment:
The new PARTIAL_COMPLETION status is a valid case of completion. The "partial" part stands for "you don't have all the data you asked for", but the job is complete and in success.
So PARTIAL_COMPLETION  shouldn't be in maybe_completed, but in completed